Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -109,6 +109,10 @@
     color : "color() * abs(${temperature} - 0.5)"
 });
 
+addStyle('Cos', {
+    color : "color() * cos(${temperature} - Math.PI)"
```

Comment:
Sorry! Will fix both of these within the next two hours. I was  unsure how else I would test cos for an expected output in the tests as there would be issues with decimal precision.